packages/app/src/cli/api/graphql/app-management/generated/create-asset-url.ts

```diff
@@ -3,7 +3,9 @@ import * as Types from './types.js'
 
 import {TypedDocumentNode as DocumentNode} from '@graphql-typed-document-node/core'
 
-export type CreateAssetUrlMutationVariables = Types.Exact<{[key: string]: never}>
+export type CreateAssetUrlMutationVariables = Types.Exact<{
+  sourceExtension: Types.SourceExtension
```

Comment:
This is coming from https://github.com/shop/world/pull/18709, so the CI will fail until it's merged.